LICENSE

```diff
@@ -0,0 +1,43 @@
+Licensed under the Open Software License version 3.0
```

Comment:
Reminder to update to whatever we decide

Reply:
Switched to ASL2: [312dcf5](https://github.com/stackabletech/hdfs-group-mapper/pull/1/commits/312dcf5dfe74bc69085286c967fa55c30b11777b)